### PR TITLE
Script: More powerful sprite creation with blender

### PR DESCRIPTION
**Pull request by heinezen**
This is a new version of our previous script. Changes:

* Completely independent from user's camera, model and animation setup
* Number of sprites per model can be specified (up to 360)
* Option for 180° mode that the Genie Engine used to have

Here is one example where the script ran on the default blender cube setup. Number of sprites was set to 64. I grouped all images together into a GIF for obvious reasons.

test_animation

### CODE REVIEW

File: scripts/blender/create_sprites.py [hunk 1]
```diff
@@ -18,43 +26,93 @@
...
+filename = bpy.path.basename(bpy.data.filepath)
+filename = filename.split('.')[0]
+
+angle = 360 / sprites
+
+# Only render 180 degrees in legacy mode
+frames_to_render = sprites
+
+if legacy_mode:
+    frames_to_render = (int)(sprites / 2) + 1
```
Comment (TheJJ) [inline scripts/blender/create_sprites.py]: there's a `int` division with `//`

File: scripts/blender/create_sprites.py [hunk 2]
```diff
@@ -18,43 +26,93 @@
...
+
+for frame in range(0, frames_to_render):
+
+    bpy.context.scene.frame_set(frame)
+
+    # File is put in a folder with the initial name
+    # of the processed blender file
+    path = '//' + filename + '/' + filename + '_' + str((int)(frame * angle))
```
Comment (TheJJ) [inline scripts/blender/create_sprites.py]: in python, int conversions are normally with `int(blabla)`

**Review by TheJJ — CHANGES_REQUESTED** [review]
great :)

**Comment by heinezen** [comment]
@TheJJ All fixed :)

**Review by TheJJ — APPROVED** [review]

**Pull request merged**